hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/source/IncrementalInputSplits.java

```diff
@@ -601,9 +602,9 @@ public List<HoodieInstant> filterInstantsWithRange(
    * @return the filtered timeline
    */
   @VisibleForTesting
-  public HoodieTimeline filterInstantsAsPerUserConfigs(HoodieTimeline timeline) {
+  public HoodieTimeline filterInstantsAsPerUserConfigs(HoodieTimeline timeline, HoodieTableType tableType) {
     final HoodieTimeline oriTimeline = timeline;
-    if (this.skipCompaction) {
+    if (OptionsResolver.isMorTable(this.conf) & this.skipCompaction) {
```

Comment:
There is no need to pass around the `HoodieTableType` now.

Reply:
Removed the para HoodieTableType.